spark/dl/src/main/scala/com/intel/analytics/bigdl/models/lenet/Train.scala

```diff
@@ -85,8 +89,9 @@ object Train {
         .setOptimMethod(optimMethod)
         .setEndWhen(Trigger.maxEpoch(param.maxEpoch))
         .optimize()
-
+      model.save(param.folder + "/model.bigdl", true)
```

Comment:
Optimizer will save the model. You should not save model like this

Reply:
Well, it is just another part of the demo. First , I used the Train method to save the model in a path and then fetch it in 'URL' form in Test and download it to another dir and load it later to compare to the original one. I think it might help show the method works.

Reply:
Don't add such demo things into the example.